src/init/features/genkit/index.ts

```diff
@@ -39,49 +39,62 @@ import {
 
 interface GenkitInfo {
   genkitVersion: string;
+  cliVersion: string;
+  vertexVersion: string;
+  googleAiVersion: string;
   templateVersion: string;
-  useInit: boolean;
   stopInstall: boolean;
 }
 
 // This is the next breaking change version past the latest template.
 const UNKNOWN_VERSION_TOO_HIGH = "2.0.0";
+const MIN_VERSION = "0.6.0";
```

Comment:
0.9.0?
we kinda technically have https://www.npmjs.com/package/genkit/v/0.6.0-dev.2

Reply:
The code didn't change. I just put it into a variable.